### DISCUSSION TITLE
User Survey V1

### DISCUSSION THREAD

**Discussion by rjheby** [PullDating/Org#119] [General]
Here's a basic version of a user Survey to start building hype and gathering research.

Looking for feedback on questions, answer options, design, etc.

Please note: we've used the maximum number of questions, which in Typeform Free is 13 questions and pages.

https://nwt5zw0sjhi.typeform.com/to/UE6yKa9J

**Comment by Oshyan** [comment]
Good start. A couple of things to add/adjust:
* I think there should be multiple choice for the first 2 questions, there's no way I could settle on just 1 choice. If you can limit it to like 3 responses, that might be good just so people don't just mark everything. 😄
* For the first question, problematic qualities, I think "too expensive" and/or "Free features are too limited" would be good additions
* The list of dating apps the person has used, there are some I've never heard of, and some that seem semi-popular not listed. So I think we should decide whether to be more comprehensive, or limit it to just the top 10 or whatever. Here's one possible source of data: https://www.businessofapps.com/data/dating-app-market/
* Also should be an "Other" option on list of dating apps with a user fill-in response
* Consider whether the order of the survey is ideal
* For the "Enter email to receive results", if you can add a note there that makes clear this does *not* mean they are signing up for any further email. I know the next question more or less makes that clear, however some people may just stop at the previous one and exit the survey if they feel like it seems spammy.
* On the final question, instead of "I don't accept", if they will still receive the survey results via email you could change it to "Please only send me a copy of my survey response" or something.

That's it from me for now. Glad this is coming together, it will be a great tool for outreach. 🙂

**Reply by rjheby** [reply]
Thanks for the feedback. I'll comment in line.

- I think there should be multiple choice for the first 2 questions, there's no way I could settle on just 1 choice. If you can limit it to like 3 responses, that might be good just so people don't just mark everything. 😄
- > done
- For the first question, problematic qualities, I think "too expensive" and/or "Free features are too limited" would be good additions
- > done
- The list of dating apps the person has used, there are some I've never heard of, and some that seem semi-popular not listed. So I think we should decide whether to be more comprehensive, or limit it to just the top 10 or whatever. Here's one possible source of data: https://www.businessofapps.com/data/dating-app-market/
- > Added everything from that list, didn't delete any
- Also should be an "Other" option on list of dating apps with a user fill-in response
- > done
- Consider whether the order of the survey is ideal
- > The idea is: Preferences, Experience, Demographics. I had sections but sections count as questions for our 13 max, so I removed them.
- For the "Enter email to receive results", if you can add a note there that makes clear this does not mean they are signing up for any further email. I know the next question more or less makes that clear, however some people may just stop at the previous one and exit the survey if they feel like it seems spammy.
- > Would prefer to leave this open ended, so we actually can market them later. 
- On the final question, instead of "I don't accept", if they will still receive the survey results via email you could change it to "Please only send me a copy of my survey response" or something.
- > I'll restrict sending survey results by email, so it's a clearer yes / no.

**Reply by rjheby** [reply]
I also added in options for "other" for the first two questions. And all the multiple choice questions, the list of answers will be randomized. 

Also, the first two questions are required. No other questions are required.
